docs/integrations/aws/index.md

```diff
@@ -32,8 +32,92 @@ In additional to runtime libraries installed by the Serverless Extension, an ext
 
-### Configuring the Serverless Console Extension
+## Configuring the Serverless Console Extension
 [Serverless Framework](../index.md) is the easiest way to instrument your app or
 service with the Serverless Console Extension. If your not using 
 Serverless Framework you'll need to build, and deploy the extension 
 yourself for each of your Lambdas functions. 
+
+### Pre-requisites
+Before you get started make sure you have the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+and a recent version of [Node.js](https://nodejs.dev/learn/how-to-install-nodejs) and a 
+[Serverless Console Organization](https://console.serverless.com?ref_website=https%3A%2F%2Fwww.serverless.com%2Fconsole%2Fdocs%2F) and [jq](https://stedolan.github.io/jq/download/).
+
+### 1.Clone the repo and install Node modules
+Once you have everything you need you'll need to clone this repo
+and install node modules.
+
+```text
+git clone https://github.com/serverless/console.git
+cd console
+npm install
+```
+
+### 2.Build the extension locally
+Next you'll need to run a set of build commands to build
+the internal and external extensions.
+
+```
+cd node/packages/aws-lambda-otel-extension/external/otel-extension-external
+npm install
+cd ../../internal/otel-extension-internal-node
+npm install
+cd ../../
+npm run build
+```
+This will generate an artifact in `dist/extension.zip`
+
+### 3.Publish the extension to your AWS Account
+Once you have built the extension, you can publish it 
+to your AWS account.
+
+```text
+aws lambda publish-layer-version \
+   --layer-name sls-otel-extension \
+   --zip-file fileb://./dist/extension.zip 
+   --region us-east-1 
+```
+
+### 4.Request an ingest token
+Next you'll need to request an ingest token based on your 
+organization id, service name, and environment you wish to
+deploy to. It's helpful to set these as environment variables.
+
+* Go to your org settings page to get your org id and org token
+* Set the following environment variables (used later)
+
+```text
+ORG_ID=<your-org-id> 
+ORG_TOKEN=<your-org-token>
+NAMESPACE=<your-namespace>
+FUNCTION_NAME<name-of-lambda-function>
+ENVIRONMENT=dev
+REGION=us-east-1
+```
+
+*  Curl the following endpoint to get an ingest token.
+```
+INGEST_TOKEN=$(curl -X POST \
+  'https://core.serverless-dev.com/ingestion/kinesis/token' \
+  --header 'Accept: application/json' \
+  --header 'sls-token-type: orgToken' \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer $ORG_TOKEN' \
+  --data-raw '{
+        "orgId": "$ORG_ID",
+        "serviceId": "test-service",
+        "stage": "$ENVIRONMENT"
+}' | jq -r .token.accessToken)
+```
+### 5.Configure necessary environment variables and tags
+
+Configure the AWS environment variables using the AWS CLI
+
+```text
+aws lambda update-function-configuration --function-name $FUNCTION_NAME --region $REGION --environment Variables= {NAMESPACE=$NAMESPACE}
```

Comment:
It doesn't feel complete, or I miss something.

Layers, as built with above instructions, require `OTEL_RESOURCE_ATTRIBUTES`, `SLS_OTEL_USER_SETTINGS` and `AWS_LAMBDA_EXEC_WRAPPER` to be configured on lambda to work.